stewards.json

```diff
@@ -70,5 +70,9 @@
     {
         "ca": "ztake_org/server_ca.crt",
         "endpoint": "https://sommelier.ztake.org:5734"
+    },
+    {
+        "ca": "MCB/server_ca.crt",
+        "endpoint": "https://sommelier.mcbnode.com:5734"
```

Comment:
I'm unable to connect to this endpoint or check the certs using the following command

```bash
openssl s_client --connect sommelier.mcbnode.com:5734 -showcerts
```